astropy/coordinates/attributes.py

```diff
@@ -52,15 +54,31 @@ class FK4(BaseCoordinateFrame):
     secondary_attribute : str
         Name of a secondary instance attribute which supplies the value if
         ``default is None`` and no value was supplied during initialization.
+    doc : str
+        Description of the frame attribute for help and documentation.
+        Information on the default value will be appended to this description.
     """
 
     name = "<unbound>"
 
-    def __init__(self, default=None, secondary_attribute=""):
+    def __init__(
+        self, default=None, secondary_attribute="", *, doc="A frame attribute"
+    ):
         self.default = default
         self.secondary_attribute = secondary_attribute
         super().__init__()
 
+        # Replace the class docstring with the custom docstring
+        self.__doc__ = doc
+
+        # Add information on the default value
+        if self.default is not None:
+            self.__doc__ += f"\n\nDefaults to {self.default}"
```

Comment:
I looked at the rendered docs and they work, but it made me realize that I think slightly better would be to replace `Defaults to` with `Default: {self.default}`.

Reply:
Done